runtime/v2/manager.go

```diff
@@ -71,9 +72,14 @@ func init() {
 			if err != nil {
 				return nil, err
 			}
+			ep, err := ic.GetByID(plugin.EventPlugin, "exchange")
+			if err != nil {
+				return nil, err
+			}
 			cs := metadata.NewContainerStore(m.(*metadata.DB))
+			events := ep.(*exchange.Exchange)
```

Comment:
I guess we should use `Publisher` here too and don't expose plugin details.
not a blocker for this PR.

Reply:
yeah, agreed